src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/lib/create_reasonable_wait.js

```diff
@@ -1,3 +1,12 @@
-export function createReasonableWait(cb) {
-  return setTimeout(cb, 100);
+export function createReasonableWait(...promises) {
+  return Promise.all([
```

Comment:
Per our conversation, I'm thinking this might be slightly more readable:
```
async function ensureMinimumTime(block, minimumTimeMs = 300) {
  // console.log(`ensureMinimumTime() minimumTimeMs=${minimumTimeMs}`);
  const start = +new Date();
  await block();
  const end = +new Date();
  const duration = end - start;

  if (duration < minimumTimeMs) {
    const timeToAdd = minimumTimeMs - (end - start);
    await new Promise(resolve => setTimeout(resolve, timeToAdd));
  }
}
```

Feel free to clean up the variable naming, etc